spec/index.bs

```diff
@@ -1721,15 +1721,14 @@ interface GPUMapMode {
             Issue(gpuweb/gpuweb#605): Handle error buffers once we have a description of the error monad.
 
             1. If |size| is unspecified:
-                1. If |offset| &gt; |this|.{{GPUBuffer/[[size]]}}, reject with an {{OperationError}} and stop.
-                1. Let |rangeSize| be |this|.{{GPUBuffer/[[size]]}} - |offset|.
+                1. Let |rangeSize| be max(0, |this|.{{GPUBuffer/[[size]]}} - |offset|).
```

Comment:
so `|rangeSize|` can end up being 0, and this will be totally valid?

Reply:
Not if `offset` is bigger than `[[size]]`

Reply:
You are specifying this as `max(0, |this|.{{GPUBuffer/[[size]]}} - |offset|)`. So if `offset` is bigger than the size, we'll get `rangeSize == 0`. What am I missing?

Reply:
Validation below says:

> - |offset| + |rangeSize| is less or equal to |this|.{{GPUBuffer/[[size]]}}

So if offset is larger than [[size]], even if rangeSize is 0, then there will be a validation error.